aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogCommitFailure.java

```diff
@@ -531,7 +531,8 @@ private int metadataFileCount(TableMetadata metadata) {
                         new File(S3TestUtil.getKeyFromUri(metadata.metadataFileLocation()))
                             .getParent())
                     .build())
-            .contents().stream()
+            .contents()
+            .stream()
```

Comment:
the formatting changes are most likely a result of updating the GJF version (which we can't update as long as we build/release with JDK8)

Reply:
> the formatting changes are most likely a result of updating the GJF version

yes, and that's why they come in separate commit

```
Update spotless to run on Java 21 

This involves code formatting changes to comply with updated formatting
library.
```


> (which we can't update as long as we build/release with JDK8)


this is discussed at https://github.com/apache/iceberg/pull/10474#discussion_r1639514171